packages/lib/gpc/src/gpcChecks.ts

```diff
@@ -538,24 +522,114 @@ export function checkVerifyClaimsForConfig(
       }
     }
   }
+  // TODO(POD-P2): List membership claims
+}
+
+/**
+ * Picks the smallest available circuit in this family which can handle the
+ * size parameters of a desired configuration.
+ *
+ * @param circuitReq the circuit size requirements
+ * @returns the circuit description, or undefined if no circuit can handle
+ *   the required parameters.
+ * @throws Error if there are no circuits satisfying the given requirements.
+ */
+export function circuitDescForRequirements(
```

Comment:
Maybe call this `pickCircuitForRequirements`?
The name `fooForBar` suggests to me a function which is doing a format conversion which would be a pure function of the input, while this one is picking from a list which is external, and doing some more complicated calculations.